### DISCUSSION TITLE
[Question]:

### DISCUSSION THREAD

**Discussion by djbetterly** [blakeblackshear/frigate#10504] [Ask A Question]
### What is your question:

I just installed Frigate for Home-Assistant via HACS, I’ve also installed MQTT.  Clearly I’m doing something wrong because when I try to launch frigate and I just get an error, “Failed to Connect”

**Comment by NickM-27** [comment]
You have only installed the integration, you must install frigate itself. https://docs.frigate.video/frigate/installation

**Comment by djbetterly** [comment]
I think I need to find a YouTube video or something, I’ve read those instructions multiple times and to me they’re very unclear for a basic user.
